packages/volto/theme/themes/pastanaga/extras/blocks.less

```diff
@@ -56,6 +56,11 @@
   background-color: rgba(120, 192, 215, 0.375);
 }
 
+.block .block.selected.error::before,
```

Comment:
Do you really need the first .block? I don't think it will make a difference if you remove it.